libnmstate/schemas/operational-state.yaml

```diff
@@ -163,6 +163,12 @@ definitions:
           minimum: 0
         flow-control:
           type: boolean
+        sr-iov:
+          type: object
+          properties:
+            total-vfs:
```

Comment:
I think you should also set the max value of total-vfs to 256.

Reply:
Done! Thanks!

Reply:
> I think you should also set the max value of total-vfs to 256.

Where does this number come from? I was wondering if there was an upper limit as well but it seemed to depend on the actual hardware. Also `256` is a strange limit, if it corresponds to an unsigned byte, the maximum would be `255` since the minimum is `0` - or does `0` indicate that there is no support and there could be `256` different VFs with IDs `0` to `255`?

Reply:
This is the only reference I could find of the 256 number.
https://www.itprotoday.com/compute-engines/q-what-sr-iov

Even that doc refers it's HW specific, so nmstate shouldn't meddle.

Sorry for misleading you @ffmancera. I think you should remove the 'maximum' .

Reply:
No problem about that. I am going to remove the `maximum`. Thanks!